courses/views/v2/__init__.py

```diff
@@ -73,12 +73,17 @@ class CourseFilterSet(django_filters.FilterSet):
 
     def filter_queryset(self, queryset):
         request = self.request
-        user = request.user if request else None
+        user = getattr(request, "user", None)
         org_id = request.query_params.get("org_id") if request else None
 
-        if not user or user.is_anonymous:
-            queryset = queryset.filter(courseruns__b2b_contract__isnull=True)
-        elif org_id:
+        show_contracted = (
```

Comment:
I think we want this to display nothing if the learner's not in the org. As is, they won't get the org's courses back but they will get everything that's public, which seems like it'd be problematic for the dashboard view. (This would be a weird state for the learner but not one that's impossible, I think.)